### PR TITLE
Dedicated path for extensions

### PR DESCRIPTION
**Pull request by akihikokuroda**
This is PR for #80

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -477,5 +477,9 @@ The backend extension is discovered by adding the label and the annotations in t
   labels:
     tekton-dashboard-extension: "true"
 ```
-Requests to the path specified in "tekton-dashboard-endpoints" annotation are routed to the extension service.
+
+1.Each extension is mounted onto /v1/extension/extension-name/ at the dashboard
+1.Requests to /v1/extension/extension-name/ are routed to / at the extension in question. / at the extension is always exposed in the expectation that many extensions will only need one endpoint.
```
[the remark this answers is not in the document]
Comment (mnuttall) [reply DEVELOPMENT.md]: Please also re-test with https://github.com/tektoncd/dashboard/pull/96 to check it still works after that's merged!

**Comment by akihikokuroda** [comment]
@a-roberts The extension path is "/v1/extension/foo" in this PR.  The Path("/v1") at line 151 prefix the "/v1" to the extension path.

File: DEVELOPMENT.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (akihikokuroda) [reply DEVELOPMENT.md]: I add the statement.

**Pull request merged**